### ISSUE TITLE
route line does not respect pathEffect and has only one (? hardcoded ?) translucent color 

### ISSUE THREAD

**Issue by chereskata** [osmandapp/OsmAnd-iOS#2165]

**Comment by PaulStets** [comment]
The route line is not the same as map objects, so path effects are not supported. It only supports the color and width (those too will soon be removed from the style as far as I know)

**Comment by chereskata** [comment]
@vshcherb But defining defaults would make some sense, to provide a sane default for a given mapstyle

**Comment by vshcherb** [comment]
Yes it makes sense but it's hard to maintain 2 options: map style and user defined, so let's see if we hard code it and let user modify it or we support rendering style defaults. As we see now most of the users customize it per profile rather than modify in the style
